petstore/clients/javascript/samples/typescript/README.md

```diff
@@ -0,0 +1,48 @@
+# client library samples for TypeScript 
+
+These sample programs show how to use the TypeScript client libraries for in some common scenarios.
+
+## Prerequisites
+
+The sample programs are compatible with [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
+
+Run server with [.NET SDK 9.0](https://dotnet.microsoft.com/en-us/download/dotnet/9.0).
+
+Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation](https://www.typescriptlang.org/docs/home.html). Install the TypeScript compiler using:
+
+
+```bash
+npm install -g typescript
+```
+
+## Setup
+
+To run the samples using the grenerated code:
+
+1. Build grenerated code in the `petstore/clients/javascript` folder:
+
+```bash
+npm install && npm run build
+```
+2. Install the dependencies using `npm` in folder `samples/javascript`:
+
+```bash
+npm install
+```
+2. Compile the samples:
+
+```bash
+npm run build
+```
+
+3. Change path to `petstore/servers/aspnet` and run the server.
+
+```bash
+dotnet run
+```
```

Comment:
we could use an absolute path so that they do not have to switch the directory like this:
```
dotnet run --project <RepoRoot>/petstore/servers/aspnet/petstore.csproj
```